packages/storybook/vue.config.js

```diff
@@ -44,6 +44,7 @@ module.exports = {
                     const updateComponentsAndTypes = [
                         'atoms',
                         'molecules',
+                        'f-status-banner',
```

Comment:
So just checking my understanding, does this mean that f-status-banner is the only `organism` that has been updated?

Reply:
Yup correct.